src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -510,18 +530,21 @@ def test_WINNF_FT_S_PCR_2(self, config_filename):
   def generate_PCR_3_default_config(self, filename):
     """Generate the WinnForum configuration for PCR 3."""
     # File path where SAS UUT claimed ppa boundary generated in PCR.1 test
-    sas_uut_claimed_ppa_boundary_file_path = os.path.join('testcases', 'output',
-                                                          'test_WINNF_FT_S_PCR_1',
-                                                          'sas_uut_claimed_ppa_boundary_'
-                                                          'default.config.json')
+    pcr_1_test_config_file_path = os.path.join('testcases', 'configs',
+                                               'test_WINNF_FT_S_PCR_1',
+                                               'default.config')
+    pcr_1_test_config_file_name = pcr_1_test_config_file_path.split('/')[-1]
+    sas_uut_claimed_ppa_boundary_file_path = getSasUutClaimedPpaBoundaryFilePath(
+        pcr_1_test_config_file_name)
```

Comment:
Can't you just use "default.config" directly?

Reply:
Based on   the suggestion we have passed only "config_file_name" only to the   function     getSasUutClaimedPpaBoundaryFilePath()